src/ray/raylet/worker_pool.h

```diff
@@ -153,7 +153,7 @@ enum class WorkerUnfitForTaskReason {
   DYNAMIC_OPTIONS_MISMATCH = 3,  // dynamic options mismatch
   OTHERS = 4,                    // reasons we don't do stats for (e.g. language)
 };
-static constexpr absl::string_view kWorkerUnfitForTaskReasonDebugName[] = {
```

Comment:
Starting from C++17, `absl::string_view` is the same as `std::string_view`.
Prefer standard library for less dependency and better stability.